.cci.jenkinsfile

```diff
@@ -7,18 +7,21 @@ parallel rpms: {
       shwrap("""
         # fetch tags so `git describe` gives a nice NEVRA when building the RPM
         git fetch origin --tags
-        ci/install-extra-builddeps.sh
-        ci/installdeps.sh
         git submodule update --init
-
-        # We lose sanitizers (all the *san) here by building straight to RPMs, but we can
-        # restore those through a build opt later on. Being able to stash RPMs directly is
-        # super nice (and archiving later on will make it easy for anyone to download
-        # binaries from PRs in the future) and meshes well with the following stages.
+        ci/installdeps.sh
+        ci/install-extra-builddeps.sh
+        # Not in the path by default
         export PATH="\$HOME/.cargo/bin:\$PATH"
-        cargo install cbindgen
-        cbindgen -c cbindgen.toml -o rpmostree-rust.h .
-
+        # Our primary CI build goes via RPM rather than direct to binaries
+        # to better test that path, including our vendored spec file, etc.
+        # However, to do that we need to generate
+        # the source code expected via the RPM path.  Because the binding
+        # build rules are part of an Automake-generated Makefile we end
+        # up running autotools twice, which is annoying but extracting
```

Comment:
Hmm looks like the sentence is cut off here.

Reply:
https://github.com/coreos/rpm-ostree/pull/2423